bentoml/_internal/cli/containerize.py

```diff
@@ -257,18 +258,15 @@ def containerize(  # type: ignore
                         output_[k] = v
                 key, value = arg.split("=")
                 output_[key] = value
-        else:
-            output_ = None
 
-        if not platform:
-            load = True
-        else:
-            if len(platform) > 1:
+        load = True
+        if platform and len(platform) > 1:
+            if not push:
                 logger.warning(
                     "Multiple `--platform` arguments were found. Make sure to also use `--push` to push images to a repository or generated images will not be saved. For more information, see https://docs.docker.com/engine/reference/commandline/buildx_build/#load."
                 )
-            else:
-                load = True
+        if push:
+            load = False
```

Comment:
Can we update the help text about the mutual exclusivity?

Reply:
will do.